src/core/tests/frontend/paddle/test_models/gen_scripts/generate_elementwise_ops.py

```diff
@@ -166,6 +188,7 @@ def elementwise_ops(name : str, data_x, data_y, axis, in_dtype):
     elementwise_add("elementwise_add" + name, data_x, data_y, axis, in_dtype)
     elementwise_sub("elementwise_sub" + name, data_x, data_y, axis, in_dtype)
     elementwise_div("elementwise_div" + name, data_x, data_y, axis, in_dtype)
+    elementwise_mul("elementwise_mod" + name, data_x, data_y, axis, in_dtype)
```

Comment:
```suggestion
    elementwise_mod("elementwise_mod" + name, data_x, data_y, axis, in_dtype)
```

Reply:
Sorry, I build openvino on remote device through ssh connection.
This error occurs when I migrate codes manual.
I've checked the remote code:
![image](https://user-images.githubusercontent.com/72954905/182992405-5f5422fb-9089-4f4b-92ba-3dddfd1d7538.png)
And update.

Reply:
Don't forget to run again the paddle fuzzy test. I'm curious whether it can pass. Because the input data type of some test case is float while elementwise_mod only support int32 or int64

Reply:
re-build and test result:
![image](https://user-images.githubusercontent.com/72954905/182994616-a24ad4e7-31d1-4c81-9751-f288e267d5ab.png)

The opset docs of FloorMod: https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/arithmetic/FloorMod_1.md
![image](https://user-images.githubusercontent.com/72954905/182994709-c5c48b92-3b44-4dda-b751-3fbe9ab08bc2.png)

Reply:
oh, I find there is a difference between paddle op cn docs and en docs:
![image](https://user-images.githubusercontent.com/72954905/182995357-639a2309-90fd-4ad6-b478-272587e3247d.png)
![image](https://user-images.githubusercontent.com/72954905/182995419-0b4f6dee-ddc5-4040-9453-969537b96162.png)